src/dotenv_vault/main.py

```diff
@@ -11,6 +11,16 @@
 import dotenv.main as dotenv
 
 
+def load_dotenv_vault() -> str:
+    path = dotenv.find_dotenv()
+    if not path:
+        return path
+    path = os.path.dirname(path)
+    if '.env.vault' not in os.listdir(path):
+        raise FileNotFoundError('.env.vault is not in same directory as .env')
```

Comment:
it shouldn't stacktrace fail. 

on local development someone might be using this library and not have a .env.vault file. that's ok. it should fallback to dotenv. 

this should have messaging like the dotenv lib:

https://github.com/motdotla/dotenv/blob/master/lib/main.js#L228

Reply:
@motdotla , okay it should default to .env